src/profile-logic/profile-store.js

```diff
@@ -24,36 +27,40 @@ export function uploadBinaryProfileData(): * {
         }
 
         xhr.onload = () => {
-          if (xhr.status === 200) {
-            resolve(xhr.responseText);
-          } else {
-            reject(
-              new Error(
-                `xhr onload with status != 200, xhr.statusText: ${
-                  xhr.statusText
-                }`
-              )
-            );
+          switch (xhr.status) {
```

Comment:
What happens in the transition state where someone is using an old client but accessing the new server? Will we get errors?

Reply:
The server URL will change anyway, so old clients will access the old server, new clients will access the new server. My plan is to keep the old server around for a few weeks.